test/internal/kubernetes/utils_create.go

```diff
@@ -336,7 +337,13 @@ func initCNSScenarioVars() (map[CNSScenario]map[corev1.OSName]cnsDetails, error)
 	cnsRoleBindingPath := cnsManifestFolder + "/rolebinding.yaml"
 	cnsServiceAccountPath := cnsManifestFolder + "/serviceaccount.yaml"
 
-	initContainerNameCNI := "acnpublic.azurecr.io/azure-cni:" + os.Getenv(envCNIVersion)
+	url, key := imageRepoURL[os.Getenv(string(envCNIImageRepo))]
+	if !key {
+		log.Printf("%s not set to expected value \"ACN\", \"MCR\". Default to %s", envCNIImageRepo, imageRepoURL["ACN"])
+		url = imageRepoURL["ACN"]
+	}
+	initContainerNameCNI := path.Join(url, "azure-cni:") + os.Getenv(envCNIVersion)
+	log.Printf("CNI init container image - %v", initContainerNameCNI)
 	initContainerNameIPAM := "acnpublic.azurecr.io/azure-ipam:" + os.Getenv(envAzureIPAMVersion)
```

Comment:
Should `azure-ipam` be pointing to the same repo ?

Reply:
I have this exact change within my cilium branch.